packages/core/src/App/Containers/Layout/header/default-mobile-links.tsx

```diff
@@ -24,17 +24,9 @@ const DefaultMobileLinks = React.memo(({ handleClickCashier }: TDefaultMobileLin
             <div className='traders-hub-header__menu-right--items--notifications'>
                 <ShowNotifications />
             </div>
-            <Popover
-                alignment='bottom'
-                classNameBubble='account-settings-toggle__tooltip'
-                message={<Localize i18n_default_text='Manage account settings' />}
```

Comment:
This DefaultMobileLinks component is rendered for mobile only, and in mobile we don't need a popover tooltip for binary link to Account Settings since there is no hover on mobile, should redirect immediately instead as per requirements.
Note: Even with this popover, it was always working for Android but not for iOS. Hence, removing to have consistent behavior across devices.